recipes/atomicrex/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "atomicrex" %}
+{% set version = "1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://gitlab.com/{{ name }}/{{ name }}/-/archive/{{ version }}/{{ name }}-{{ version }}.tar.gz
+  sha256: f90d7a46987a363c2d14d693e7ac1d0e966b8a0a852aafe3254b30f006ea6555
+  patches:
+    - conda.patch
+
+build:
+  number: 0
+  skip: true  # [not linux]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('fortran') }}
+    - make
+    - cmake 
+  host:
+    - python
+    - boost-cpp
+    - libxml2
+    - muparser
+    - nlopt
+    - vim
```

Comment:
Just because I haven’t seen it before, is this needed in host? Glancing at the source it looks like it is used as part of the build system and should be in build?